docs/dom-testing-library/api-queries.md

```diff
@@ -220,8 +224,10 @@ If it is important that you query an actual `<label>` element you can provide a
 const inputNode = screen.getByLabelText('Username', { selector: 'input' })
 ```
 
-Note that it will not work in the case where a `for` attr on a `label` matches
-and `id` field on a non-form element.
+> **Note**
+>
+> `getByLabelText` will not work in the case where a `for` attribute on a
+> `<label>` element matches an `id` attribute on a non-form element.
```

Comment:
I've changed it to read: "a label matches **an** `id` field" instead of **and**. I also removed "it" in favour of explicitly referring to `getByLabelText` and updated references to `attr` and `field` to be `attribute`.